float/src/main/java/com/novoda/floatschedule/convert/GithubUserConverter.java

```diff
@@ -0,0 +1,13 @@
+package com.novoda.floatschedule.convert;
+
+import java.util.List;
+
+public interface GithubUserConverter {
+    
+    List<String> getGithubUsers() throws Exception;
```

Comment:
i agree, but my issue here was with supporting the pre-existent user converter:
```
public String getFloatUser(String githubUsername) throws IOException, NoMatchFoundException
```
i'd rather have just the no match exception but `readIfNeeded()` throws the `IOException` and the other we throw ourselves.